src/cloud-api-adaptor/cmd/cloud-api-adaptor/main.go

```diff
@@ -137,10 +143,15 @@ func (cfg *daemonConfig) Setup() (cmd.Starter, error) {
 		if err != nil {
 			return nil, fmt.Errorf("secure comms failed to initialize KubeMgr: %w", err)
 		}
+		if strings.EqualFold(secureCommsKbsAddr, "false") {
```

Comment:
Strike that. So when secureComms is `true` and trustee is not there (aka Attestation Phase is optional), how is that handled?

Reply:
secureComms bool flag control if secureComms is enabled or not.

Currently the PR does not have a dedicated  bool flag to indicate "No attestation" mode. 
Instead, the PR reuse the `secure-comms-kbs` string flag. If that flag is set to "false", the inner `cfg.serverConfig.SecureCommsKbsAddress` is set to an empty string which is used internally to indicate that "No attestation" mode.

We can instead add a bool flag such as `secureCommsSkipAttestation`.

Reply:
How about using empty string for `secure-comms-kbs` to infer no attestation is needed? A valid value will indicate attestation is required.

Reply:
I would prefer not to do that since this will not allow us to use the default "kbs-service.trustee-operator-system:8080" which are are using today. 
Today, if we `secure-comms-kbs` is not set, it is set to this default value which makes sense to anyone installing the system against a standard trustee-operator deployment.

Reply:
I see.. In that case using a separate bool flag makes more sense and is intuitive. 
@stevenhorsman @yoheiueda wdyt? Should skipping attestation phase be a separate bool flag or setting the secure-comms-kbs addr to `false` is fine ?

Reply:
A separate bool flag sounds more intuitive to me.

Reply:
We can add this is a separate PR

Reply:
@stevenhorsman @yoheiueda do you have any preference w.r.to adding the bool flag in this PR versus a new PR? Please see the thread above for context

Reply:
I have now added a NoTrustee flag to replace setting the SecureCommsKbsAddress to false.